File: Leetcode/319-Bulb-Switcher/BulbSwitcher.js
/**
 * @param {number} n
 * @return {number}
 */
var bulbSwitch = function(n) {
    if (n<1) {
        return 0
    }
    var res = 1
    while (res*res <=n) {
        res +=1
    }
    return res-1
};

File: Leetcode/319-Bulb-Switcher/BulbSwitcher.java
public class BulbSwitcher {
    public int bulbSwitch(int n) {
        if (n<1) {
            return 0;
        }
        int res = 1;
        while (res*res <=n) {
            res++;
        }
        return res -1;
    }// end of bulbSwitch       
}






